packages/plugin-dom-editable/src/EventNormalizer.ts

```diff
@@ -1611,9 +1630,10 @@ export class EventNormalizer {
         // Don't trigger events on the editable if the click was done outside of
         // the editable itself or on something else than an element.
         const target = this._getEventTarget(ev);
-        if (target && this._isInEditable(target)) {
+        const caretPosition = this._getEventCaretPosition(ev);
+        if (target && caretPosition && this._isInEditable(caretPosition.offsetNode)) {
```

Comment:
caretPosition will always be an object so no need to check it with `&& caretPosition`